purchase_order_migration/models/purchase_order.py

```diff
@@ -0,0 +1,10 @@
+# Copyright 2024 Quartile Limited
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from odoo import fields, models
+
+
+class PurchaseOrder(models.Model):
+    _inherit = "purchase.order"
+
+    old_id = fields.Integer()
```

Comment:
Shouldn't this be in a separate module if we eventually uninstall this module once the migration is done?